https://github.com/mtp401/protoc-grpcio/pull/42

diff --git a/Cargo.toml b/Cargo.toml
index 9f785e09..5c616cfa 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -19,8 +19,8 @@ documentation = "https://docs.rs/protoc-grpcio"
 keywords = ["compiler", "grpc", "protobuf", "grpc-rs"]
 license = "Apache-2.0"
 repository = "https://github.com/mtp401/protoc-grpcio"
-[dependencies.failure]
-version = "0.1"
+[dependencies.anyhow]
+version = "1"

 [dependencies.grpcio-compiler]
 version = "0.6"
diff --git a/src/lib.rs b/src/lib.rs
index 3b38ba57..8ed4ea37 100644
--- a/src/lib.rs
+++ b/src/lib.rs
@@ -34,7 +34,7 @@
 extern crate grpcio_compiler;

 #[macro_use]
-extern crate failure;
+extern crate anyhow;

 extern crate tempfile;

@@ -49,16 +49,16 @@ use std::iter::Iterator;
 use std::path::{Path, PathBuf};
 use std::vec::Vec;

-use failure::ResultExt;
+use anyhow::Context;

 use tempfile::NamedTempFile;

-use protobuf::{compiler_plugin, descriptor};
+use protobuf::{compiler_plugin, descriptor, Message};
 use protobuf_codegen::Customize;
 use protoc::{DescriptorSetOutArgs, Protoc};

 /// Custom error type used throughout this crate.
-pub type CompileError = ::failure::Error;
+pub type CompileError = ::anyhow::Error;
 /// Custom result type used throughout this crate.
 pub type CompileResult<T> = Result<T, CompileError>;

@@ -253,7 +253,7 @@ where
         .context("failed to read descriptor set")?;

     let descriptor_set =
-        protobuf::parse_from_bytes::<descriptor::FileDescriptorSet>(&serialized_descriptor_set)
+        descriptor::FileDescriptorSet::parse_from_bytes(&serialized_descriptor_set)
             .context("failed to parse descriptor set")?;

     let customize = customizations.unwrap_or_default();
